configuration/esapi/ESAPI.properties

```diff
@@ -407,6 +407,10 @@ Logger.UserInfo=true
 # Determines whether ESAPI should log the session id and client IP.
 Logger.ClientInfo=true
 
+# Determines whether ESAPI should log the prefix of [EVENT_TYPE - APPLICATION NAME].
+# If all above Logger entries are set to false and LogIgnorePrefix is true, then the output would be the same like if no ESAPI was used
+Logger.LogIgnorePrefix=true
```

Comment:
The boolean logic / variable name is a little confusing.

I think the logic will be easier to manage if we phrase this as a Positive statement:
```
# Determines whether ESAPI should log the prefix of [EVENT_TYPE - APPLICATION NAME].
Logger.LogPrefix=true  
```

I'd prefer to remove the additional comment from line 411, as that may lead to confusion around the "other log entries" if (and when) the properties are expanded.

Changing the naming to a positive statement will also apply throughout the naming of the implementation, and the  boolean logic checks.